src/main/java/com/minecolonies/util/StructureInfo.java

```diff
@@ -0,0 +1,90 @@
+package com.minecolonies.util;
+
+import java.nio.file.Path;
+import java.util.*;
+
+/**
+ * Created by Dars on 9/21/2016.
+ */
+public class StructureInfo
+{
+
+    /**
+     * The name this schematic has.
+     */
+    private String name;
+
+    /**
+     * The category  of the structure.
+     */
+    private String category = "hut";
+
+    private Boolean isHut = false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'isHut' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)